custom/icds_reports/models/aggregate.py

```diff
@@ -891,3 +893,47 @@ def compare_with_old_data(cls, state_id, month):
             cursor.execute(query, params)
             rows = fetchall_as_namedtuple(cursor)
             return [row.child_health_case_id for row in rows]
+
+
+class AggregateInactiveAWW(models.Model):
+    awc_id = models.TextField(primary_key=True)
+    awc_name = models.TextField(blank=True, null=True)
+    awc_site_code = models.TextField(blank=True, null=True)
+    supervisor_id = models.TextField(blank=True, null=True)
+    supervisor_name = models.TextField(blank=True, null=True)
+    block_id = models.TextField(blank=True, null=True)
+    block_name = models.TextField(blank=True, null=True)
+    district_id = models.TextField(blank=True, null=True)
+    district_name = models.TextField(blank=True, null=True)
+    state_id = models.TextField(blank=True, null=True)
+    state_name = models.TextField(blank=True, null=True)
+    first_submission = models.DateField(blank=True, null=True)
+    last_submission = models.DateField(blank=True, null=True)
+
+    @property
+    def days_since_start(self):
+        if self.first_submission:
+            delta = date.today() - self.first_submission
+            return delta.days
+        return 'N/A'
+
+    @property
+    def days_inactive(self):
+        if self.last_submission:
+            delta = date.today() - self.last_submission
+            return delta.days
+        return 'N/A'
+
+    @classmethod
+    def aggregate(cls, last_sync):
+        helper = InactiveAwwsAggregationHelper(last_sync)
+        missing_location_query = helper.missing_location_query()
+        aggregation_query, agg_params = helper.aggregate_query()
+
+        with get_cursor(cls) as cursor:
+            cursor.execute(missing_location_query)
+            cursor.execute(aggregation_query, agg_params)
+
+    class Meta(object):
+        app_label = 'icds_reports'
+        db_table = AGG_INACTIVE_AWW_TABLE
```

Comment:
These don't need to have manually specified db_tables. That's an anti-pattern that's been needed where we have been manually controlling databases with raw sql